docs/adr/002-single-binary-assets.md

```
@@ -0,0 +1,178 @@
+Encode CSS as base64 data URLs in link href.
+
+**Pros:**
+- No code changes to serving logic
+
+**Cons:**
+- 33% larger (base64 overhead)
+- Ugly URLs in HTML source
+
+## Decision
+
+**Recommended: Option B (Inline CSS)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)